pkg/deploy/che_configmap.go

```diff
@@ -73,7 +73,6 @@ func GetCustomConfigMapData() (cheEnv map[string]string) {
 		"CHE_INFRA_KUBERNETES_SERVICE__ACCOUNT__NAME":           "che-workspace",
 		"CHE_WORKSPACE_AUTO_START":                              "true",
 		"CHE_INFRA_KUBERNETES_WORKSPACE__UNRECOVERABLE__EVENTS": "FailedMount,FailedScheduling,MountVolume.SetUp failed,Failed to pull image",
-		"CHE_LIMITS_WORKSPACE_IDLE_TIMEOUT":                     "-1",
```

Comment:
I'm afraid that this change won't be applied to already deployed Che where custom config map is already created.
If it won't - should we care about existing installation where the limit will be still -1?
Do we really need put any configuration here if we are not able to propagate new defaults? I predict that default `UNRECOVERABLE__EVENTS` will be changed soon as well... So, maybe it would be better to create empty custom config map?
cc @davidfestal

Reply:
yes, it will only be effective for new installations, until issue https://github.com/eclipse/che/issues/14635 is implemented.